source-mongodb/main.go

```diff
@@ -97,9 +96,7 @@ func (c *config) ToURI() string {
 
 	uri.User = url.UserPassword(c.User, c.Password)
 
-	if c.Database != "" {
-		uri.Path = "/" + c.Database
-	}
+	uri.Path = "/"
```

Comment:
nit: Remove this line entirely?

Reply:
@williamhbaker I actually initially did, but MongoDB client complains about this, as it seems to add its own `?query=` parameters, and if there is no slash, it considers the URI to be invalid without a slash:

```
connecting to database: error parsing uri: must have a / before the query ?
```